devTools/check_trailing_whitespaces.sh

```diff
@@ -28,6 +28,7 @@ files_with_trailing_whitespaces=$(
         -type f \
         -not -name "*.cache" \
         -not -name "*.log" \
+        -not -path "./.DS_Store" \
```

Comment:
is it really related?

Reply:
well, implicitly, without this `make autoreview-docker` doesn't pass